crates/pathfinder/src/core.rs

```diff
@@ -41,6 +41,25 @@ pub struct ContractCode {
     pub abi: String,
 }
 
+/// A Starknet contract's definition.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct ContractDefinition {
```

Comment:
I would had preferred that this, and the Program were defined as "Lite" or whatever prefix, and commented that this is not the full contract definition. This is just to get the ABI and code.

Does the abi: String imply the raw value automatically or is this some other representation? We shouldn't be passing it as a string, that'd escape all of the json right?

Reply:
* "Lite" prefix - good idea :+1:
* It's not really used for any deserialization, we should remove `Deserialize` from derives here imho (and from `ConractCode` too).
* Btw, we should remove `Serialize` from their `sequencer::reply` counterparts.

Reply:
Yep, will add a comment to explain why String and why not Deserialize.

Reply:
Prefix added, derives removed, docs added.